scripts/anno_dl.sh

```diff
@@ -0,0 +1,58 @@
+# Adapted from coco_dl.sh
+# Removed the image datasets, they are not required for creating dataframe objects
```

Comment:
I'm wondering if this script can be merged with the existing `coco_dl.sh` script. It seems all we need to add is another command line argument after dataset path. (From first glance I don't see anything new in this script except for the removal of the image zip download)

Reply:
Per our discussion this afternoon I've wrapped the image downloading portion in `coco_dl.sh`. Passing `no_image` as the second argument prevents the script from downloading the image dataset.